src/main/build/debug.h

```diff
@@ -122,6 +122,7 @@ typedef enum {
     DEBUG_TPA,
     DEBUG_S_TERM,
     DEBUG_SPA,
+    DEBUG_ALTHOLD,
     DEBUG_TASK,
```

Comment:
Needs to be at the end.
```suggestion
    DEBUG_TASK,
    DEBUG_ALTHOLD,
```

Reply:
thanks

Reply:
Done!